dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/FfdcCleanupTest.java

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.logging.fat;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.ProtocolException;
+import java.net.URL;
+
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.RemoteFile;
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.config.Logging;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+
+import componenttest.annotation.ExpectedFFDC;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+import componenttest.topology.utils.HttpUtils;
+
+/**
+ *
+ */
+@RunWith(FATRunner.class)
+public class FfdcCleanupTest {
+    private static final String SERVER_NAME_XML = "com.ibm.ws.logging.ffdcCleanup";
+    private static final int CONN_TIMEOUT = 10;
+    private static LibertyServer server;
+
+    @BeforeClass
+    public static void initialSetup() throws Exception {
+        server = LibertyServerFactory.getLibertyServer(SERVER_NAME_XML);
+        server.saveServerConfiguration();
+        ShrinkHelper.defaultDropinApp(server, "ffdc-servlet", "com.ibm.ws.logging.fat.ffdc.servlet");
+
+    }
+
+    public void setUp() throws Exception {
```

Comment:
Add `@Before` will make this method run before each test case starts so you can skip calling this in each test case.

Reply:
Ah yes thank you, I was originally setting up several servers but merged them into one and forgot to fix this.